src/components/Activity/ActivityProvider.js

```diff
@@ -43,6 +48,25 @@ function getStoredList(account) {
       status: SymbolsByName.get(`ACTIVITY_STATUS_${activity.status}`),
     }),
   })
+
+  // Temporary function to reset activities if package version changed
```

Comment:
This is a bit coupled to this component; I wonder if we should use a more explicit mechanism where we can detail "migrations" like this since it's not just wiping the entire cache every time.

For example, on starting up the app, we check between last and current version and do any associated "migrations" within those ranges.